vk_video_decoder/libs/VkDecoderUtils/FFmpegDemuxer.cpp

```diff
@@ -308,6 +308,9 @@ class FFmpegDemuxer : public VideoStreamDemuxer {
     #ifdef VK_EXT_video_decode_vp9
         case AV_CODEC_ID_VP9        : return VK_VIDEO_CODEC_OPERATION_DECODE_VP9_BIT_KHR;
     #endif // VK_EXT_video_decode_vp9
+    #ifdef vulkan_video_codec_av1std_decode
+		case AV_CODEC_ID_AV1        : return VK_VIDEO_CODEC_OPERATION_DECODE_AV1_BIT_KHR;
```

Comment:
Misaligned fields. Somebody needs to fix their editor to use spaces and not tabs.